templates/integration/tsconfig.json.ejs

```diff
@@ -8,5 +8,5 @@
     "module": "esnext",
     "moduleResolution": "node"
   },
-  "include": ["src"]
+  "include": ["src", ".spectral/*"]
```

Comment:
```suggestion
  "include": ["src", ".spectral"]
```
Believe we only need the dir name here, like src.

Reply:
We need `*` for this to work.